<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RstPad::SearchReplaceDlg</class>
 <widget class="QDialog" name="RstPad::SearchReplaceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>256</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Search / replace</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLineEdit" name="Text">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Text to find</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Replacement">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Replacement</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="MatchingBox">
         <property name="title">
          <string>Matching</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="CaseSensitive">
            <property name="text">
             <string>case sensitive</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="WholeWords">
            <property name="text">
             <string>whole words</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="Wildcards">
            <property name="toolTip">
             <string>? - any one character, * - any string</string>
            </property>
            <property name="text">
             <string>wildcards</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="DirectionBox">
         <property name="title">
          <string>Search direction</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="DirectionForward">
            <property name="text">
             <string>forward</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="DirectionBackward">
            <property name="text">
             <string>backward</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="Find">
       <property name="text">
        <string>Find</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Replace">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ReplaceAll">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Replace all</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Count">
       <property name="text">
        <string>Count</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Text</tabstop>
  <tabstop>Replacement</tabstop>
  <tabstop>Find</tabstop>
  <tabstop>Replace</tabstop>
  <tabstop>ReplaceAll</tabstop>
  <tabstop>Count</tabstop>
  <tabstop>Cancel</tabstop>
  <tabstop>CaseSensitive</tabstop>
  <tabstop>WholeWords</tabstop>
  <tabstop>Wildcards</tabstop>
  <tabstop>DirectionForward</tabstop>
  <tabstop>DirectionBackward</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
